stories/Components/UIcomponents/Cards/BioCard/BioCard.jsx

```diff
@@ -42,9 +42,7 @@ export const BioCard = ({
       <div className="grid-x grid-margin-x">
         <div className={['cell', `${size_options[`${args.size}`]}`].join(' ')}>
           {args.size === 'large' && (
-            <a
-              id="bio-card"
-              className={['grid-x grid-margin-x bio-card', `${hover_color_options[`${args.hovercolor}`]}`].join(' ')}
+            <a className={['grid-x grid-margin-x bio-card', `${hover_color_options[`${args.hovercolor}`]}`].join(' ')}
```

Comment:
@pallavikadam-srijan There is whitespace in class attribute. Kindly use the provided solution of `cls` to add/toggle classes.